app/models/mixins/archived_mixin.rb

```diff
@@ -2,11 +2,14 @@ module ArchivedMixin
   extend ActiveSupport::Concern
 
   included do
+    scope :deleted, -> { where(:deleted => true) }
+    scope :not_deleted, -> { where(:deleted => false) }
+
     belongs_to :old_ext_management_system, :foreign_key => :old_ems_id, :class_name => 'ExtManagementSystem'
   end
 
   def archived?
-    ems_id.nil?
+    deleted?
```

Comment:
I see vms have `archived?` (ems_id and storage_id nil), `orphaned?` (ems_id nil, storage_id not nil), a `not_archived_nor_orphaned`, plus related virtual attributes and *_CONDITIONS sql consts.
Hosts have simple `archived?` for ems_id nil.
These terms also appear in UI.

Oh, terminology-wise we also have "disconnect".